src/Core/Models/Mail/BaseMailModel.cs

```diff
@@ -18,5 +18,12 @@ public string WebVaultUrlHostname
                 return WebVaultUrl;
             }
         }
+        public string CurrentYear
+        {
+            get
+            {
+                return DateTime.Now.Year.ToString();
```

Comment:
Seems inconsequential, however for server-apps should always use `DateTime.UtcNow` vs. `Now`; where we're only getting the year it doesn't matter as much, but for consistency sake `UtcNow` is more appropriate IMO.

Reply:
Fair! Will update for consistency.

Reply:
Updated.